docker/sentinel-compose.yml

```
@@ -0,0 +1,98 @@
+  redis-server-1:
+    image: falkordb/falkordb:edge
+    hostname: redis-server-1
+    container_name: redis-server-1
+    ports:
+      - 6380:6380
+    networks:
+      - redis-sentinel
+    entrypoint: ["/bin/bash", "-c"]
+    command: |
+      touch /data/node.conf
+      echo "port 6380" >> /data/node.conf 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)